src/parser/csql_grammar.y

```diff
@@ -24133,25 +24135,85 @@ dblink_column_definition
 
 opt_access_modifier
     : /*empty*/
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, : );
 
 	$$ = PT_PRIVATE;
 
       DBG_PRINT}}
     | PRIVATE
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, | PRIVATE);
 
 	$$ = PT_PRIVATE;
 
       DBG_PRINT}}
     | PUBLIC
-      {{
+      {{ DBG_TRACE_GRAMMAR(opt_access_modifier, | PUBLIC);
 
 	$$ = PT_PUBLIC;
 
       DBG_PRINT}}
     ;
 
+/*
+ * To prevent the change of the error message below,
+ * the rules of opt_or_replace and opt_access_modifier are combined into opt_create_synonym and used.
+ * 
+ * Query: CREATE PARTITION TABLE;
+ * - AS-IS: Syntax error: unexpected 'partition', expecting CLASS or TABLE
+ * - TO-BE: Syntax error: unexpected 'partition', expecting SYNONYM
```

Comment:
Did you change it to show 'CLASS or TABLE'?
The contents of the comments and Jira's comments are different.
```
AS-IS
csql> create partition table;
 *** Token) [create]
 *** Rule) stmt::= :
 *** Token) [ ]
 *** Token) [partition]
 *** Rule) create_stmt::= : CREATE
 *** Rule) opt_hint_list::= :

In line 1, column 8,

ERROR: In line 1, column 8 before ' table; '
Syntax error: unexpected 'partition', expecting CLASS or TABLE

TO-BE (NOT WANT)
csql> create partition table;
 *** Token) [create]
 *** Rule) stmt::= :
 *** Token) [ ]
 *** Token) [partition]
 *** Rule) opt_or_replace::= :
 *** Rule) opt_access_modifier::= :

In line 1, column 8,

ERROR: In line 1, column 8 before ' table; '
Syntax error: unexpected 'partition', expecting SYNONYM
```

Reply:
This is a change to prevent it from being changed to TO-BE.
```
TO-BE (NOT WANT)
```